README.md

```diff
@@ -34,9 +34,21 @@ Bashhub can be accessed from the command line in a couple ways:
 - `bashhub` for everything else
 
 It also provides a key binding of `ctrl + b` for quickly dropping into interactive search.
+
 ## Search
 You can search through your commands in a lot of different ways. Check `bh --help` for more specifics. By default `bh` will output the most recently used unique commands to standard out. Adding the `-i` argument to any `bh` search will make it **interactive**. There are also several arguments to pass to filter, query, and get more specific about your searches!
 
+### Search with Fuzzy Finder
```

Comment:
I don't think this belongs at the top of the search section here. This section's goal is to detail ways to search with `bh`, hence the examples right below. Including anything else at the top here is a little off-topic. 

Search with Fuzzy Finder is a nice simple addition that we can include at the end of the section :)

Reply:
I think down down around line 112, right above the `Bashhub Status` section would be fine.